src/g3log/time.hpp

```diff
@@ -55,6 +56,15 @@ namespace g3 {
    * std::put_time. A possible fix if your c++11 library is not updated is to
    * modify this to use std::strftime instead */
    std::string localtime_formatted(const system_time_point& ts, const std::string& time_format) ;
+
+   inline system_time_point to_system_time(const high_resolution_time_point& ts)
+   {
+	   using namespace std::chrono;
+	   static const auto hrs_now = high_resolution_clock::now();
```

Comment:
@AndreasSchoenle I've updated to correct the other minor issues that I found with this pull request. 
Could you please here add a comment to explain to the code reader why the time adjustment is done?

Once that is done I think this is ready to be merged. I've tested it on 3 platforms.